IdentityCore/src/throttling/cache/MSIDThrottlingCacheRecord.h

```diff
@@ -20,21 +20,21 @@
 // AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-// THE SOFTWARE.  
+// THE SOFTWARE.
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface MSIDThrottlingCacheRecord : NSObject
 
 @property (nonatomic, readonly) NSDate *creationTime;
 @property (nonatomic, readonly) NSDate *expirationTime;
-@property (nonatomic, readonly) NSString *throttleType;
+@property (nonatomic) NSUInteger throttleType;
```

Comment:
just curious why this is not readonly? are we expecting this to change for the lifetime of the record object?